package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "dyson-front-end-react-template",
+  "private": true,
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "test": "vitest run",
+    "test:coverage": "vitest run --coverage",
```

The difference here is that one of this runs is for CI and the other is for TDD. 
For TDD we should `vitest --coverage`. Then tests are run in watch mode and produce coverage information needed for developer. 
For CI we need to run tests once with coverage directory generated `vitest run`
```suggestion
    "test": "vitest --coverage",
    "test:ci": "vitest run",
```

---

done thank you :-)